Neos.Media/Classes/Domain/Service/ImageService.php

```diff
@@ -141,6 +141,9 @@ public function processImage(PersistentResource $originalResource, array $adjust
             $imagineImage->usePalette(new RGB());
         }
 
+        $stripFilter = new \Imagine\Filter\Basic\Strip();
+        $stripFilter->apply($imagineImage);
```

Comment:
Does this strip all metadata from an image? 

I'm asking because hard coding this would then probably make this package: `https://github.com/neos/metadata-extractor` or am I mistaken?

Reply:
This reduces also the size of the file. Not much, but, if we can take it… 😄